diff --git a/godot/player/player.tscn b/godot/player/player.tscn
index e02f915..ddfe0a9 100644
--- a/godot/player/player.tscn
+++ b/godot/player/player.tscn
@@ -77,6 +77,11 @@ xfade_time = 0.1
 advance_mode = 2
 advance_expression = "is_on_floor()"
 
+[sub_resource type="AnimationNodeStateMachineTransition" id="Trans_fall_jump"]
+xfade_time = 0.05
+advance_mode = 2
+advance_expression = "not is_on_floor() and velocity.y > 0.0"
+
 [sub_resource type="AnimationNodeStateMachineTransition" id="Trans_jump_ground"]
 xfade_time = 0.05
 advance_mode = 2
@@ -90,7 +95,7 @@ states/jump/node = SubResource("14")
 states/jump/position = Vector2(250, -50)
 states/fall/node = SubResource("15")
 states/fall/position = Vector2(250, 250)
-transitions = [&"Start", &"ground", SubResource("Trans_start_ground"), &"ground", &"jump", SubResource("Trans_ground_jump"), &"ground", &"fall", SubResource("Trans_ground_fall"), &"jump", &"fall", SubResource("Trans_jump_fall"), &"fall", &"ground", SubResource("Trans_fall_ground"), &"jump", &"ground", SubResource("Trans_jump_ground")]
+transitions = [&"Start", &"ground", SubResource("Trans_start_ground"), &"ground", &"jump", SubResource("Trans_ground_jump"), &"ground", &"fall", SubResource("Trans_ground_fall"), &"jump", &"fall", SubResource("Trans_jump_fall"), &"fall", &"ground", SubResource("Trans_fall_ground"), &"fall", &"jump", SubResource("Trans_fall_jump"), &"jump", &"ground", SubResource("Trans_jump_ground")]
 
 [sub_resource type="AnimationNodeBlendTree" id="23"]
 graph_offset = Vector2(-221.364, -194.029)
