--- upstream/project.godot	2026-09-10 14:24:28
+++ walker/project.godot	2026-09-24 00:44:45
@@ -10,7 +10,7 @@
 
 [application]
 
-config/name="Dodge the Creeps"
+config/name="Walker — Dodge the Creeps"
 config/description="This is a simple game where your character must move
 and avoid the enemies for as long as possible.
 
--- upstream/player.gd	2026-09-10 14:24:28
+++ walker/player.gd	2026-09-24 01:15:29
@@ -31,6 +31,7 @@
 	position = position.clamp(Vector2.ZERO, screen_size)
 
 	if velocity.x != 0:
+		rotation = 0
 		$AnimatedSprite2D.animation = &"right"
 		$AnimatedSprite2D.flip_v = false
 		$Trail.rotation = 0
