diff --git a/godot/player/weapon/Sword.tscn b/godot/player/weapon/Sword.tscn
index d9b35d7..e1096b5 100644
--- a/godot/player/weapon/Sword.tscn
+++ b/godot/player/weapon/Sword.tscn
@@ -163,6 +163,18 @@ tracks/3/keys = {
 "method": &"set_ready_for_next_attack"
 }]
 }
+tracks/4/type = "value"
+tracks/4/imported = false
+tracks/4/enabled = true
+tracks/4/path = NodePath(".:monitoring")
+tracks/4/interp = 1
+tracks/4/loop_wrap = true
+tracks/4/keys = {
+"times": PackedFloat32Array(0, 0.15),
+"transitions": PackedFloat32Array(1, 1),
+"update": 1,
+"values": [true, false]
+}
 
 [sub_resource type="Animation" id="4"]
 resource_name = "attack_medium"
@@ -204,6 +216,18 @@ tracks/2/keys = {
 "update": 1,
 "values": [true]
 }
+tracks/3/type = "value"
+tracks/3/imported = false
+tracks/3/enabled = true
+tracks/3/path = NodePath(".:monitoring")
+tracks/3/interp = 1
+tracks/3/loop_wrap = true
+tracks/3/keys = {
+"times": PackedFloat32Array(0.05, 0.25),
+"transitions": PackedFloat32Array(1, 1),
+"update": 1,
+"values": [true, false]
+}
 
 [sub_resource type="Animation" id="5"]
 length = 0.01
diff --git a/godot/player/weapon/sword.gd b/godot/player/weapon/sword.gd
index afc9744..b068c19 100644
--- a/godot/player/weapon/sword.gd
+++ b/godot/player/weapon/sword.gd
@@ -63,7 +63,6 @@ func _change_state(new_state: States) -> void:
 			attack_current = combo[combo_count -1]
 			$AnimationPlayer.play(attack_current["animation"])
 			visible = true
-			monitoring = true
 
 	state = new_state
 
