diff --git a/godot/player/weapon/sword.gd b/godot/player/weapon/sword.gd
index b068c19..919e183 100644
--- a/godot/player/weapon/sword.gd
+++ b/godot/player/weapon/sword.gd
@@ -62,6 +62,7 @@ func _change_state(new_state: States) -> void:
 		States.ATTACK:
 			attack_current = combo[combo_count -1]
 			$AnimationPlayer.play(attack_current["animation"])
+			$AnimationPlayer.seek(0.0)
 			visible = true
 
 	state = new_state
